packages/cli/src/commands/init/editTemplate.ts

```diff
@@ -106,3 +106,26 @@ export function changePlaceholderInTemplate({
       processDotfiles(filePath);
     });
 }
+
+export function changeReactNativeVersionInTemplate(newVersion: string) {
+  const packageJsonPath = path.join(process.cwd(), './package.json');
```

Comment:
We have access to `directory` from the callsite of this function. Let's use that instead of `process.cwd` which we try to avoid